torch/_torch_docs.py

```diff
@@ -9739,6 +9739,7 @@ def merge_dicts(*dicts):
     unitriangular (bool, optional): whether :math:`A` is unit triangular.
         If True, the diagonal elements of :math:`A` are assumed to be
         1 and not referenced from :math:`A`. Default: ``False``.
+    out ((Tensor, Tensor), optional): the output tensors
```

Comment:
To properly document the out parameter it should also appear in the signature.

"out" is a Keyword arg, which we typically separate from the "Args"

Reply:
thanks, i should have looked at some other examples first!